src/main/java/edu/ithaca/dturnbull/bank/Admin.java

```diff
@@ -0,0 +1,30 @@
+package edu.ithaca.dturnbull.bank;
+public class Admin extends AbstractBank {
```

Comment:
Should this really extend AbstractBank. Seems like Admin knows about Abstract Bank.